edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/AgentEventIds.cs

```diff
@@ -29,6 +29,9 @@ public struct AgentEventIds
         public const int ModuleClientProvider = EventIdStart + 2300;
         public const int LogsRequestHandler = EventIdStart + 2400;
         public const int TaskStatusRequestHandler = EventIdStart + 2500;
+        public const int KubernetesPlanner = EventIdStart + 2600;
+        public const int KubernetesOperator = EventIdStart + 2700;
+        public const int KubernetesCommand = EventIdStart + 2800;
```

Comment:
It was suggested to me to move these out of Core in my PR, I did, and there's a KubernetesEventIds.cs in the Kubernetes project,  You should probably also move these to that file.